src/components/form/textinput/TextInputElement.tsx

```diff
@@ -75,19 +80,19 @@ export default class TextInputElement extends React.Component<TextInputProps> {
     }
 
     return (
-      <FormElement
-        __className={this.props.__className}
-        name={this.props.name}
-        helpText={this.props.helpText}
-        showLabel={this.props.showLabel}
-        // errors={this.state.errors}
-        entry={this.props.entry}
-      >
-        <div className={styles.wrapper + ' ' + styles[this.props.style || TextInputStyle.normal]}>
-          {createTextInput(this.props, this.handleChange, optional)}
-          {charCount}
-        </div>
-      </FormElement>
+      <>
+        <UnnnicTextArea
```

Comment:
All text inputs will become text-areas?
What is the behavior in other card types like Call External service and Open a ticket for example?

Reply:
This component only modifies the textarea components